nodes.toml

```
@@ -1,93 +1,74 @@
 [sourceNode]
-node = "http://localhost:26657"
+node = "http://sentry-01.hosts.testnet.cudos.org:26657"
 
-totalNodesChecked = 79
+totalNodesChecked = 27
 
 successfulNodes = {
-    "http://141.94.138.48:26677": "1535001",
-    "http://65.21.24.47:26657": "3561653",
-    "http://104.245.147.210:26657": "3486001",
-    "http://3.76.85.22:17658": "3366001",
-    "http://65.21.202.61:25005": "3574369",
-    "http://65.108.127.249:2001": "1961001",
-    "http://162.19.170.233:15457": "3560001",
-    "http://65.109.91.165:2001": "3399904",
-    "http://104.245.147.209:26657": "3693001",
-    "http://localhost:26657": "3382001",
+    "http://185.247.207.13:26657": "3603400",
```

in proper TOML, we don't use colons.
```suggestion
    "http://185.247.207.13:26657" = "3603400",
```

---

but that is just a note, please instead do as described in slack.